### PR TITLE
obs: qemu-vanilla: point to qemu repository.

### PR DESCRIPTION
**Pull request by jcvenegas**
Lets point to qemu repository instead of kata fork for qemu-vanilla.

The qemu vanilla sha was was incorrect, fix it.

Fixes: #161

Signed-off-by: Jose Carlos Venegas Munoz
<jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

**Comment by ydjainopensource** [comment]
@nitkon Looking at the [hash](https://git.qemu.org/git.qemu.org/qemu.git/?p=qemu.git;a=commit;h=0982a56a551556c704dc15752dabf57b4be1c640) it looks like its qemu version 2.11.2

**Comment by jodh-intel** [comment]
\o/

lgtm

File: obs-packaging/gen_versions_txt.sh [hunk 1]
```diff
@@ -73,7 +73,7 @@ qemu_lite_version=${qemu_lite_version}
 qemu_lite_hash=${qemu_lite_hash}
```
Comment (devimc) [inline obs-packaging/gen_versions_txt.sh]: please fix commit message: The qemu vanilla sha *was was* incorrect.

Comment (jcvenegas) [reply obs-packaging/gen_versions_txt.sh]: @devimc , thanks. Fixed now.

**Comment by jcvenegas** [comment]
@ydjainopensource  @nitkon at the moment  https://github.com/kata-containers/packaging/blob/master/obs-packaging/gen_versions_txt.sh is going to check  https://github.com/kata-containers/runtime/blob/master/versions.yaml  (today suggest the branch `version: "stable-2.11"`. `gen_versions_txt.sh` will take the latest release from 2.11.x

File: obs-packaging/qemu-vanilla/_service-template [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <services>
  <service name="tar_scm">
     <param name="scm">git</param>
-    <param name="url">https://github.com/kata-containers/qemu.git</param>
+    <param name="url">https://github.com/qemu/qemu.git</param>
```
Comment (devimc) [inline obs-packaging/qemu-vanilla/_service-template]: do you know if the `checkout` to the version 2.11 is automatic?

Comment (jcvenegas) [reply obs-packaging/qemu-vanilla/_service-template]: kind of the checkout is don by the xml node.
` <param name="revision">@QEMU_VANILLA_HASH@</param>`

Comment (devimc) [reply obs-packaging/qemu-vanilla/_service-template]: make sense, thanks

**Pull request merged**